test/eth_getUncles.js

```diff
@@ -0,0 +1,113 @@
+const assert = require("assert");
+const Web3 = require("web3");
+const { provider } = require("../index");
```

Comment:
Use:

```
var Ganache = require(process.env.TEST_BUILD
  ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
  : "../index.js");
```

here and all other tests that import ganache-core so we can also run our tests against the webpacked version of the library.